1.11/storage/persistent-volume/index.md

```diff
@@ -151,29 +208,29 @@ The temporary Mesos sandbox is still the target for the `stdout` and `stderr` lo
 
 # Examples
 
-## Run stateful PostgreSQL on Marathon
+## Stateful PostgreSQL on Marathon
+
+A model app definition for PostgreSQL on Marathon would look like the following. Note that we set the PostgreSQL data folder to `pgdata`, which is relative to the Mesos sandbox (as contained in the `$MESOS_SANDBOX` variable). This enables us to set up a persistent volume with a `containerPath` of `pgdata`. This path is is not nested and relative to the sandbox as required:
 
-A model app definition for PostgreSQL on Marathon would look like this. Note that we set the PostgreSQL data folder to `pgdata`, which is relative to the Mesos sandbox (as contained in the `$MESOS_SANDBOX` variable). This enables us to set up a persistent volume with a `containerPath` of `pgdata`. This path is is not nested and relative to the sandbox as required:
 
 ```json
 {
   "id": "/postgres",
   "cpus": 1,
   "instances": 1,
   "mem": 512,
-  "networks": [ 
-    { 
-      "mode": "container/bridge" 
-    } 
-  ],
+  "networks": [ { "mode": "container/bridge" } ],
```

Comment:
Probably the formatting is slightly inconsistent here. I am referring to the spaces between brackets, and between brackets and strings.